docs/datamodel/constraints.rst

```diff
@@ -164,6 +164,11 @@ The standard library defines the following constraints:
             }
         }
 
+    This constraint also has an additional effect of creating an
+    implicit :ref:`index <ref_datamodel_indexes>` on the link or
+    property. This means that in the above example there's no need to
+    add explicit indexes for the ``name`` property or the ``owns``
```

Comment:
Indexes on links are not necessary, they're always created implicitly.

Reply:
OK, I've updated the docs to reflect that,